test/hardhat/MetaMorpho.spec.ts

```diff
@@ -13,7 +13,7 @@ import {
 } from "@nomicfoundation/hardhat-network-helpers/dist/src/helpers/time";
 
 // Must use relative import path.
-import SpeedJumpIrmArtifact from "../../lib/morpho-blue-irm/out/SpeedJumpIrm.sol/SpeedJumpIrm.json";
+// import SpeedJumpIrmArtifact from "../../lib/morpho-blue-irm/out/SpeedJumpIrm.sol/SpeedJumpIrm.json";
```

Comment:
Do you want to keep this as a comment ?

Reply:
This could be removed indeed, because it is planned to revert those commits in the long term anyway